examples/nextjs/pages/customize.tsx

```diff
@@ -80,14 +80,23 @@ export function Stage() {
       <div className={styles.participantGrid}>
         <GridLayout tracks={tracks}>
           <TrackRefContext.Consumer>
-            {(track) =>
-              track && (
+            {(trackRef) =>
+              trackRef && (
                 <div className="my-tile">
-                  {isTrackReference(track) ? <VideoTrack {...track} /> : <p>Camera placeholder</p>}
+                  {isTrackReference(trackRef) ? (
+                    <VideoTrack trackRef={trackRef} />
+                  ) : (
+                    <p>Camera placeholder</p>
+                  )}
                   <div className={myStyles['participant-indicators']}>
                     <div style={{ display: 'flex' }}>
-                      <TrackMutedIndicator source={Track.Source.Microphone} />
-                      <TrackMutedIndicator source={track.source} />
+                      <TrackMutedIndicator
+                        trackRef={{
+                          participant: trackRef.participant,
+                          source: Track.Source.Microphone,
```

Comment:
this feels a bit cumbersome with track references. not sure if trackReference is the right approach for this